cmd/release-controller-api/http_helper.go

```diff
@@ -902,5 +904,13 @@ func checkConsistentImages(release, parent *releasecontroller.Release) ReleaseCh
 		sort.Strings(errDoesNotExist)
 		result.Errors = append(result.Errors, fmt.Sprintf("Found recent tags downstream that are not built upstream (not in %s/%s): %s", parent.Source.Namespace, parent.Source.Name, strings.Join(errDoesNotExist, ", ")))
 	}
+	if len(warnOlderThanUpstream) > 0 {
+		sort.Strings(warnOlderThanUpstream)
+		result.Warnings = append(result.Warnings, fmt.Sprintf("Some tags are significantly older than the upstream: %s", strings.Join(warnOlderThanUpstream, ", ")))
+	}
+	if len(warnNoDownstream) > 0 {
+		sort.Strings(warnNoDownstream)
+		result.Warnings = append(result.Warnings, fmt.Sprintf("No downstream builds have been pushed for these upstream tags: %s", strings.Join(warnNoDownstream, ", ")))
+	}
```

Comment:
@jupierce removed these lines in #379, on March 8th.  Not sure how/why you are adding them back...

Reply:
Hmm, sorry, I am not sure how. It certainly should not be there. Fixed it.